crates/uv/src/commands/toolchain/list.rs

```diff
@@ -64,7 +64,7 @@ pub(crate) async fn list(
         result
             .as_ref()
             .err()
-            .map_or(true, DiscoveryError::is_critical)
```

Comment:
Wait this one is `map_or(true)`

Reply:
Oops, fixed. Need `is_none_or` to stabilize to refactor those.